packages/xorg_lib.rb

```diff
@@ -23,5 +39,7 @@ class Xorg_lib < Package
   depends_on 'libxvmc'
   depends_on 'libxxf86dga'
   depends_on 'libxxf86vm'
+  depends_on 'libxcomposite'
+  depends_on 'libxscrnsaver'

```

Comment:
Should we sort these to avoid duplicates or does the order matter?

Reply:
No duplicates, I have tested it with fresh install.

Reply:
This kind of list might help us to fix previous error. I can know which packages I have worked on.

Reply:
I was just thinking the list is so long it would be easy to accidentally add a duplicate but if they were sorted, it would be a bit easier to avoid.